feedthefox/devices/templates/jinja2/devices.html

```diff
@@ -9,32 +9,32 @@
 
     <div class="row">
       <div class="col-sm-3">
-        <form>
+        <form action="" method="GET">
           <div class="form-group">
             <label>Manufacturer</label>
-            <select class="form-control" name="manufacturer">
-              <option value="0" selected>All</option>
-              <option value=""></option>
-            </select>
+            {{ field_with_attrs(device_filter.form.manufacturer, class="form-control") }}
           </div>
           <div class="form-group">
             <label>Type</label>
-            <select class="form-control" name="type">
-              <option value="0" selected>Smartphone</option>
-            </select>
+            {{ field_with_attrs(device_filter.form.type, class="form-control") }}
```

Comment:
I get that, all I 'm saying is that since we don't have other options for now, lets query either against everything or against `smartphone` and hide the drop-down box from the user. Since we don't have any other option for now, I am in favor of simplicity. We can add the field in the html when we have more options.